communities/activity-event-and-media-recommendations.instructor.md

```diff
@@ -0,0 +1,32 @@
+# Instructor: Activity: Event and Media Recommendations
+
+Presentations are exciting because it gives students an opportunity to verbalize what gets them excited.
+
+Getting the slides to auto-advance every 30 seconds is important. This helps moderate the student pace; no student should "hijack" the presentation, or take up too much space. Auto-advance helps prevent facilitation.
```

Comment:
The last sentence is confusing to me. Isn't facilitation a good thing? Why would we want it prevented? My guess is there's a typo here, or some otherwise crossed wire, but I'm not sure what the intent was. Maybe just remove "prevent"?
```suggestion
Getting the slides to auto-advance every 30 seconds is important. This helps moderate the student pace; no student should "hijack" the presentation, or take up too much space. Auto-advance helps facilitation.
```

Reply:
Yeah, my brain was probably working too quickly. I meant something like... prevent a need for manual facilitation over timing. (I've seen our very polite facilitators (including me) needing to work through cutting someone off who's presenting over time! It's hard!)

Thanks!